.github/workflows/resolve_merge_conflict.yml

```
@@ -0,0 +1,16 @@
+name: 'Resolve Merge Conflicts'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  resolve-merge-conflicts:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.13.x
+
+    - name: Build app
```

```suggestion
    - name: Run build.sh
```

No name is really needed, so just omitting it would also be OK.